pallets/node-authorization/src/tests.rs

```diff
@@ -392,3 +392,23 @@ fn get_authorized_nodes_works() {
 		);
 	});
 }
+
+#[test]
+fn add_connection_already_connected_error() {
```

Comment:
nit: Change name to standard `adding_already_connected_connection_should_fail`

Reply:
done.